app/models/twitter_account.rb

```diff
@@ -0,0 +1,78 @@
+class TwitterAccount < ActiveRecord::Base
+  scope :active, -> { where(active: true) }
+  scope :national, -> { active.where(national: true) }
+
+  has_many :tweets, dependent: :destroy
+
+  serialize :twitter_account_info
```

Comment:
Any reason why you do this rather than using a json (or jsonb) column? I believe json is more performant, and you can query the db for matching attributes rather than needing to instantiate...

Reply:
Ahhh. Because it's a direct port from the existing service. I still think it would be worth switching to jsonb here.

Reply:
Good call thanks — the text column is vestigial from the original implementation. Updated in 6ae1d74